packages/drivers/odsp-driver/src/contracts.ts

```diff
@@ -34,6 +34,11 @@ export interface ISocketStorageDiscovery {
      * passed as a parameter to `OdspDocumentService.create()` factory.
      */
     socketToken?: string;
+
+    /**
+     * This is the time within which client has to refresh the session on push.
```

Comment:
"on push" is hard to understand.
I switched to using "(ODSP) relay (ordering) service" everywhere. That's the language I think we are standardizing on